tests/unit/Utils/DPOPTest.php

```diff
@@ -416,15 +438,16 @@ final public function testGetWebIdWithDpopWithoutThumbprint(): void
 
         $dpop->getWebId($request);
     }
-
+*/
     /**
      * @testdox Dpop SHOULD complain WHEN asked to get WebId from Request with valid DPOP with Thumbprint not matching Key Id
-     *
+     * why? which spec?
```

Comment:
I think this and the previous tests were part of the KID confusion. We'll have to come back to them later in more detail.

Also: Instead of commenting out the entire function, it is enough to change the function name to not start with `test`. (for instance `_test` or `skipTest`).

Or, better yet... place [`$this->markTestSkipped()`](https://phpunit.readthedocs.io/en/9.5/incomplete-and-skipped-tests.html#skipping-tests) with a sensible message at the top of the test.